src/components/MessagesList/MessagesList.vue

```diff
@@ -763,6 +853,25 @@ export default {
 			}, 500)
 		},
 
+		checkSticky() {
+			const ulElements = this.$el.querySelectorAll('.dates-group-wrapper')
```

Comment:
Check how refs array is used in LeftSidebar:
``:ref="`conversation-${item.token}`"``

In general, code looks good, I just don't know if we could avoid using `classList.add` here

Reply:
We can use its dateTimestamp and store it in a data prop and then 
`:class = {"is-sticky" : this.stickyId = dateTimestamp`

so to avoid `classList.add`

Reply:
Good idea, there should be only one sticky element